demo-server/templates/index-template.handlebars

```diff
@@ -76,7 +76,9 @@
         function initWidget(widget) {
           {{#if redirectless}}
           var config = {
-            // interactWith: PfAuthnWidget.INTERACT_WITH_USER_AUTHZ,
+            // flowType: PfAuthnWidget.FLOW_TYPE_USER_AUTHZ,
```

Comment:
Should we make clear that `flowType` and `user_code` only apply to the device authorization case?